pkgs/shells/fish/plugins/fifc.nix

```diff
@@ -1,23 +1,37 @@
 {
   lib,
+  pkgs,
```

Comment:
> Thanks for the correction. But `fzf` has the same as the fish plugin [`fzf`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/shells/fish/plugins/default.nix#L33), so may be add `pkgs-fzf` as the real fzf to `plugins/default.nix`?

why not `fishPlugins.fzf` and `fzf`

Reply:
`fishPlugins.fzf` is in scope `pkgs.fishPlugins`, `fzf` is in scope `pkgs`, unless you introduce `pkgs.fishPlugins` scope, or it will always be `pkgs.fzf`

Reply:
see https://github.com/NixOS/nixpkgs/blob/3e0bedf9638ba1d46fd51d4a484fa78b8a97562c/pkgs/top-level/splice.nix#L137-L140

Reply:
For scope that `callPackage` auto filled in arguments, see: https://github.com/NixOS/nixpkgs/blob/3e0bedf9638ba1d46fd51d4a484fa78b8a97562c/pkgs/top-level/splice.nix#L137-L140

Reply:
Try this. It's a common pattern.
```nix
  fifc = callPackage ./fifc.nix {
    inherit (pkgs) fzf;
  };
```

Reply:
Sorry, I don't understand what you mean. Did you try my suggestion?